libs/langchain/tests/unit_tests/conftest.py

```diff
@@ -110,11 +110,7 @@ def test_something():
                 # If we haven't yet checked whether the pkg is installed
                 # let's check it and store the result.
                 if pkg not in required_pkgs_info:
-                    try:
-                        installed = util.find_spec(pkg) is not None
-                    except Exception:
```

Comment:
This was added [here](https://github.com/langchain-ai/langchain/pull/14373#pullrequestreview-1770960480).
But is it really useful ?